tools/c7n_salactus/c7n_salactus/cli.py

```diff
@@ -216,6 +222,7 @@ def invoke(f, *args, **kw):
                 account_info['buckets'] = bucket
             if region:
                 account_info['regions'] = region
+            account_info['id-map'] = id_map
```

Comment:
so for each account storing the same map over again?

Reply:
It's used as a lookup later on in `process_key_chunk`

Reply:
i understand the lookup, its just that this is a big value thats being duplicated n times, instead of stored once.